src/python/WMComponent/TaskArchiver/CleanCouchPoller.py

```diff
@@ -161,8 +161,9 @@ def algorithm(self, parameters=None):
         try:
             logging.info("Creating workload summary")
             finishedwfsWithLogCollectAndCleanUp = DataCache.getFinishedWorkflows()
+            logging.info("Total %s workload summary will be created", len(finishedwfsWithLogCollectAndCleanUp))
             self.archiveSummaryAndPublishToDashBoard(finishedwfsWithLogCollectAndCleanUp)
-            logging.info("%s workload summary is created", len(finishedwfsWithLogCollectAndCleanUp))
+            logging.info("All workload summary docs were uploaded")
```

Comment:
uploaded or created? Don't we upload them somewhere down the chain?

Reply:
Actually we are uploading the summary in workload_summary couch db here.